src/app/components/identifier/identifier.component.html

```diff
@@ -0,0 +1,24 @@
+<div [class.has-list-class]="listClass">
+    <evt-ms-desc-section [label]="'place'" [additionalClass]="'settlement'" [inlineLabel]="true">
```

Comment:
The label is displayed even if it has no value.
Example in https://www.tei-c.org/release/doc/tei-p5-doc/en/html/examples-altIdentifier.html